action/store/postgres.py

```diff
@@ -48,8 +51,9 @@ def read(self) -> dict:
                 pgc.relispartition = false
                 and pgc.relkind in ('r', 'v', 'm', 'p')
                 and c.table_schema not in ('information_schema', 'pg_catalog') 
-                and c.table_name not in ('migrations')
-            ;"""
+                and (cardinality(%(exclude)s::text[]) = 0 or c.table_name != any(%(exclude)s))
```

Comment:
Can you switch the two "or" conditions? Second one is more readable, so makes it more obvious what the whole line is trying to do. What case is the cardinality condition accounting for?

Reply:
done

Reply:
the cardinality condition is to account for when `exclude` is empty.

`SELECT 'foo' != ANY('{}')`yields `FALSE`

Reply:
That is a very cryptic statement. Consider replacing with either `not in`, I believe that should work even with an empty set, or format the query string in Python to include/exclude that condition in the SQL depending on whether the exclude list is empty.